Update links to use broad hg38 bundle

arahuja opened the pull request:
The older links were broken, this uses the GATK provided bundle, but this is technically hg38 (meaning the contig are `chr` prefixed.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/hammerlab/biokepi/308)
<!-- Reviewable:end -->

smondet commented on src/run_environment/reference_genome.ml:
```diff
@@ -144,11 +144,9 @@ module Default = struct
+      "ftp://gsapubftp-anonymous@ftp.broadinstitute.org/bundle/hg38/hg38bundle/dbsnp_144.hg38.vcf.gz" in
     let gtf_b38_url =
       "http://ftp.ensembl.org/pub/release-79/gtf/homo_sapiens/\
        Homo_sapiens.GRCh38.79.gtf.gz" in
```
Isn't the GTF going to be using the wrong chronames?

Wouldn't it be better to create a whole new ref-genome `hg38`?

arahuja replied:
Yea that makes sense. I was hoping to avoid that since we will end up with a split - no hg38 GTF, can't find a b38 dbSNP etc.

arahuja commented:
I updated this to have a separate `h38` reference and:
- removed broken dbsnp link from b38
- added a known indels arg to reference builds as well and added this to hg19, b37 and h38
